From 1e4ebb7b4d3ecc716b25535b99e2f0d71b13605a Mon Sep 17 00:00:00 2001
Message-Id: <1e4ebb7b4d3ecc716b25535b99e2f0d71b13605a.1398812351.git.mark.gross@intel.com>
In-Reply-To: <e87e3533536ae5eec0e06ceb7ef7428e42855790.1398812350.git.mark.gross@intel.com>
References: <e87e3533536ae5eec0e06ceb7ef7428e42855790.1398812350.git.mark.gross@intel.com>
From: Heikki Krogerus <heikki.krogerus@linux.intel.com>
Date: Mon, 17 Feb 2014 15:43:16 +0200
Subject: [PATCH 238/429] ACPI / platform: add IDs for Broadcom Bluetooth chip

These IDs are used on Baytrail boards such as Lenovo Miix 2
and Asus Transformer Book T100TA. The actual chip should be
BCM43241 with all these IDs.

Signed-off-by: Heikki Krogerus <heikki.krogerus@linux.intel.com>
---
 drivers/acpi/acpi_platform.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/acpi/acpi_platform.c b/drivers/acpi/acpi_platform.c
index 0187753..9967493 100644
--- a/drivers/acpi/acpi_platform.c
+++ b/drivers/acpi/acpi_platform.c
@@ -33,6 +33,9 @@ static const struct acpi_device_id acpi_platform_device_ids[] = {
 	{ "VPC2004" },
 	{ "BCM4752" },
 	{ "LNV4752" },
+	{ "BCM2E1A" },
+	{ "BCM2E39" },
+	{ "BCM2E3D" },
 
 	/* Intel Smart Sound Technology */
 	{ "INT33C8" },
-- 
1.8.3.2

